util/src/main/java/io/zeebe/util/sched/ActorTask.java

```diff
@@ -294,9 +294,7 @@ public void onFailure(final Throwable failure) {
         break;
 
       default:
-        Loggers.ACTOR_LOGGER.error(
-            "Actor failed in phase '{}'. Continue with next job.", lifecyclePhase, failure);
-
+        actor.handleFailure(failure);
```

Comment:
So the actor can decide whether he wants to fail ?

Reply:
Yes. Previous behavior is that actor never fails and continue with next job. So I kept it as the default behavior. The actors (mostly the ones that allows health check) can then decide to fail.